scripts/users_scrapper.ts

```diff
@@ -0,0 +1,48 @@
+import { DateTime } from 'luxon'
+import logger from '@adonisjs/core/services/logger'
+import db from '@adonisjs/lucid/services/db'
+
+const url = 'https://live.pwr.edu.pl/sks/sks-data.csv'
+
+export async function runScrapper() {
+  try {
+    const response = await fetch(url)
+    if (!response.ok) {
+      logger.error(`Failed to fetch data: ${response.status} ${response.statusText}`)
+      return
+    }
+
+    const usersData = await response.text()
+    const currentDateTime = DateTime.now().setZone('Europe/Warsaw')
+
+    const rows = usersData.trim().split('\n')
+
+    const values = rows
+      .map((row) => {
+        const [time, movingAverage21, activeUsers] = row.split(';')
+        const timestamp = currentDateTime.set({
+          hour: Number.parseInt(time.split(':')[0], 10),
+          minute: Number.parseInt(time.split(':')[1], 10),
+          second: 0,
+          millisecond: 0,
+        })
+
+        return `('${timestamp}', ${Number(activeUsers)}, ${Number(movingAverage21)}, NOW(), NOW())`
```

Comment:
just a note that it's not sanitized so possibly an sql injection is possible here

Reply:
sql injections is something to always have in mind when doing raw queries with dynamically generated strings

Reply:
I know the only person that can threat us is Mr. Tomasz from Informatyzacja, but still it should be pretty easy to do? I hope so, didn't read about it in lucid/adonis